New MCP2221 requirement of hidapi causing problems in other repos

When doing a release of PyBadger, Travis failed because hidapi would not install because the command `sudo apt-get install libudev-dev libusb-1.0` was not run on that repo. I'm unsure of the best solution, but here's a couple of possibilities:

* Put some kind of conditional in requirements.txt and setup.py so that hidapi is only included for MCP2221
* Add `sudo apt-get install libudev-dev libusb-1.0` to the .travis.yml of all other repos as they fail

I think the first option would be best, but I'm not sure if it's possible. I figured I'd open this in case anybody else has ideas.

Also, as reference, here's the error with deploying PyBadger:
```
Building wheels for collected packages: Adafruit-Blinka, Adafruit-PlatformDetect, Adafruit-PureIO, pyftdi, hidapi, sysv-ipc, spidev, pyusb
  Building wheel for Adafruit-Blinka (setup.py): started
  Building wheel for Adafruit-Blinka (setup.py): finished with status 'done'
  Stored in directory: /home/travis/.cache/pip/wheels/da/18/80/a9156e455b46b88adfedc41c9d640dc12dde2da46bb9aa47c2
  Building wheel for Adafruit-PlatformDetect (setup.py): started
  Building wheel for Adafruit-PlatformDetect (setup.py): finished with status 'done'
  Stored in directory: /home/travis/.cache/pip/wheels/77/c0/06/edb6710d07f1552f7fb7d8cb7e991d2b952c6ce67de20131eb
  Building wheel for Adafruit-PureIO (setup.py): started
  Building wheel for Adafruit-PureIO (setup.py): finished with status 'done'
  Stored in directory: /home/travis/.cache/pip/wheels/7e/9f/e0/a4851d6f3a5fd21b05ff529b69206baabb8b8ec510dab94639
  Building wheel for pyftdi (setup.py): started
  Building wheel for pyftdi (setup.py): finished with status 'done'
  Stored in directory: /home/travis/.cache/pip/wheels/cc/eb/83/16ad5d23b6ac860ca098cdc535004b3d6b6276bdbe427ce321
  Building wheel for hidapi (setup.py): started
  Building wheel for hidapi (setup.py): finished with status 'error'
  Complete output from command /home/travis/virtualenv/python3.6.7/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-vhntteqz/hidapi/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-2iuofsv0 --python-tag cp36:
  running bdist_wheel
  running build
  running build_ext
  cythoning hid.pyx to hid.c
  /tmp/pip-install-vhntteqz/hidapi/.eggs/Cython-0.29.14-py3.6-linux-x86_64.egg/Cython/Compiler/Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: /tmp/pip-install-vhntteqz/hidapi/hid.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)
  cythoning hidraw.pyx to hidraw.c
  /tmp/pip-install-vhntteqz/hidapi/.eggs/Cython-0.29.14-py3.6-linux-x86_64.egg/Cython/Compiler/Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: /tmp/pip-install-vhntteqz/hidapi/hidraw.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)
  building 'hid' extension
  creating build
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/hidapi
  creating build/temp.linux-x86_64-3.6/hidapi/libusb
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -fPIC -Ihidapi/hidapi -I/usr/include/libusb-1.0 -I/opt/python/3.6.7/include/python3.6m -c hid.c -o build/temp.linux-x86_64-3.6/hid.o
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -fPIC -Ihidapi/hidapi -I/usr/include/libusb-1.0 -I/opt/python/3.6.7/include/python3.6m -c hidapi/libusb/hid.c -o build/temp.linux-x86_64-3.6/hidapi/libusb/hid.o
  hidapi/libusb/hid.c:47:20: fatal error: libusb.h: No such file or directory
  compilation terminated.
  error: command 'gcc' failed with exit status 1
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -6,5 +6,4 @@ rpi_ws281x>=4.0.0; platform_machine=='armv7l' or platform_machine=='armv6l'
 spidev>=3.4; sys_platform == 'linux'
 sysv_ipc; sys_platform == 'linux'
 pyftdi>=0.30.0
-hidapi
 binho-host-adapter>=0.1.4

